src/Symfony/Bundle/FrameworkBundle/composer.json

```
@@ -28,6 +28,7 @@
         "symfony/http-kernel": "^5.4|^6.0",
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php80": "^1.16",
+        "symfony/polyfill-php81": "^1.16",
```

```suggestion
        "symfony/polyfill-php81": "^1.22",
```

The function that we need was introduced with 1.22.0. See symfony/polyfill#324